addons/mail/static/src/messaging/entity/thread/thread.js

```diff
@@ -418,6 +443,47 @@ function ThreadFactory({ Entity }) {
             }
         }
 
+        /**
+         * Open a dialog to add channels as followers.
+         */
+        promptAddChannelFollower() {
+            this._promptAddFollower({ mail_invite_follower_channel_only: true });
+        }
+
+        /**
+         * Open a dialog to add partners as followers.
+         */
+        promptAddPartnerFollower() {
+            this._promptAddFollower({ mail_invite_follower_channel_only: false });
+        }
+
+        /**
+         * Refresh followers information from server.
+         */
+        async refreshFollowers() {
+            const [{ message_follower_ids: followerIds }] = await this.env.rpc({
+                model: this.model,
+                method: 'read',
+                args: [this.id, ['message_follower_ids']]
+            });
+            if (followerIds && followerIds.length > 0) {
+                const { followers } = await this.env.rpc({
```

Comment:
Probably discussed already but shouldn't make 2 following rpc ideally.

Reply:
I think it should be done later because it needs changes python-side or on the way form view works

Reply:
Ok for doing it later, please add a "fixme" and create related task.

Reply:
https://www.odoo.com/web#id=2243180&action=333&active_id=1519&model=project.task&view_type=form&cids=1&menu_id=4720